### PR TITLE
calling out that Huawei app is out of date

### PR DESCRIPTION
**Pull request by DanStevensonMSFT**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/get-teams-android-in-china.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ If you're in China, use the links in this article to get Microsoft Teams for And
 |---------|---------|
 | Lenovo: https://aka.ms/TeamsLenovo      | ![QR code for Teams on Android from Lenovo store.](media/get-teams-android-in-china-lenovo.png)        |
 | Xiaomi: https://aka.ms/TeamsXiaomi     |![QR code for Teams on Android from Xiaomi store.](media/get-teams-android-in-china-xiaomi.png)         |
-|Huawei: https://aka.ms/TeamsHuawei     | ![QR code for Teams on Android from Huawei store.](media/get-teams-android-in-china-huawei.png)        |
+|Huawei: https://aka.ms/TeamsHuawei  NOTE: The Teams app in the Huawei app store is several months out of date and missing many features and security updates. Microsoft recommends that Huawei customers use a different phone or update the app from a different app store.   | ![QR code for Teams on Android from Huawei store.](media/get-teams-android-in-china-huawei.png)        |
```
Comment (JohanFreelancer9) [inline Teams/get-teams-android-in-china.md:29]:
```suggestion
|Huawei: https://aka.ms/TeamsHuawei  **NOTE**: The Teams app in the Huawei app store is several months out of date and missing many features and security updates. We recommend that Huawei customers update the app from a different app store.   | ![QR code for Teams on Android from Huawei store.](media/get-teams-android-in-china-huawei.png)        |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @DanStevensonMSFT, kindly commit the suggested changes if you agree with them. Many thanks.

**Comment by DanStevensonMSFT** [comment]
copy edits look great

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
This file, get-teams-android-in-china, has been removed. Closing this PR.

**Pull request closed without merging**